packages/core/src/components/AutocompleteField/AutocompleteField.jsx

```diff
@@ -0,0 +1,198 @@
+}
+
+/**
+ * An `AutocompleteField` component renders a typeahead input field, form label, and a clear input link.
+ */
+export class AutocompleteField extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.id = uniqueId('autocomplete_');
+  }
+
+  renderChildren(getInputProps) {
+    // Extend props on the TextField, by passing them through
+    // Downshift's `getInputProps` method
+    return React.Children.map(this.props.children, child => {
+      const { disabled } = this.props;
+
+      if (isTextField(child)) {
+        return React.cloneElement(
+          child,
+          getInputProps({
+            className: disabled ? 'ds-c-autocomplete__inner--disabled' : '',
+            disabled: disabled,
+            fieldClassName: 'ds-c-autocomplete__input',
+            id: this.id,
+            inputId: this.id,
+            labelClassName: 'ds-u-margin-top--0 ds-c-autocomplete__label'
+          })
+        );
+      }
+
+      return child;
+    });
+  }
+
+  render() {
+    const {
+      clearAriaLabel,
+      clearInputText,
+      constrainedList,
+      constrainedListText,
+      items,
+      itemToString,
+      onChange,
+      onStateChange
+    } = this.props;
+
+    return (
+      <Downshift
+        itemToString={itemToString}
+        onChange={onChange}
+        onStateChange={onStateChange}
+        render={({
+          clearSelection,
+          getInputProps,
+          getItemProps,
+          highlightedIndex,
+          isOpen,
+          inputValue
+        }) => (
+          <div className="ds-u-clearfix ds-c-autocomplete">
+            {this.renderChildren(getInputProps)}
+
+            {isOpen ? (
+              <div className="ds-u-border--1 ds-u-padding--1 ds-c-autocomplete__list">
+                {constrainedList ? (
+                  <h5
+                    className="ds-u-margin--0 ds-u-padding--1"
+                    id="constrained-list-header"
+                  >
+                    {constrainedListText}
+                  </h5>
+                ) : null}
+
+                <ul
+                  aria-labelledby={
+                    constrainedList ? 'constrained-list-header' : null
+                  }
+                  className="ds-c-list--bare"
+                  role="listbox"
+                >
+                  {items
+                    .filter(
+                      item =>
+                        !inputValue ||
+                        item.name
+                          .toLowerCase()
+                          .includes(inputValue.toLowerCase())
+                    )
+                    .map((item, index) => (
+                      <li
+                        aria-selected={highlightedIndex === index}
+                        className={
+                          highlightedIndex === index
+                            ? 'ds-u-padding--1 ds-c-autocomplete__list-item ds-c-autocomplete__list-item--active'
```

Comment:
We could place the padding within the `ds-c-autocomplete__list-item` CSS declaration to remove the need for a utility class here

Reply:
Done. Added the padding to `.autocomplete__list-item` using `$spacer-2` variable.